internal/venice-common/src/main/java/com/linkedin/venice/common/VeniceSystemStoreUtils.java

```diff
@@ -12,6 +14,12 @@ public class VeniceSystemStoreUtils {
   private static final String PUSH_JOB_DETAILS_STORE_NAME =
       String.format(Store.SYSTEM_STORE_FORMAT, PUSH_JOB_DETAILS_STORE);
   public static final String SEPARATOR = "_";
+  public static final UpdateStoreQueryParams DEFAULT_USER_SYSTEM_STORE_UPDATE_QUERY_PARAMS =
```

Comment:
This static field is the same as the one defined in the following class, so can reuse it?

Reply:
Yes they're duplicate. I'll remove one of them.